conf/config.neon

```diff
@@ -23,6 +23,8 @@ parameters:
 	reportMagicProperties: false
 	ignoreErrors: []
 	internalErrorsCountLimit: 50
+	cachedNodesByFileCountMax: 512
+	cachedNodesByStringCountMax: 512
```

Comment:
I'd rather have nested cache: section.

You can reference nested parameters by %cache.xxx%.

Reply:
Agree